MUXSDKStats/MUXSDKStats/MUXSDKStats.h

```diff
@@ -37,7 +37,7 @@ FOUNDATION_EXPORT
 + (_Null_unspecified instancetype)new NS_UNAVAILABLE;
 
 /*!
- @method      monitorAVPlayerViewController:withPlayerName:playerData:videoData:
+ @method      monitorAVPlayerViewController:withUrl:withPlayerName:playerData:videoData:
```

Comment:
clean up the doc lines on these now that we didn't change the signature

Reply:
cleaned